### PR TITLE
add Mastodon and Rumble

### PR DESCRIPTION
**Pull request by PalmTreeVI**

### CODE REVIEW

**Review by Liquid369 — DISMISSED** [review]
ACK 90e5ba47839e0bbb0a020179bcd10187ca02b0ca

File: 06.community/article.md [hunk 1]
```diff
@@ -42,8 +42,10 @@ Ordered by popularity:
 * Facebook: https://www.facebook.com/PIVXCrypto
 * Reddit: https://reddit.com/r/pivx
 * Linkedin: https://ca.linkedin.com/company/pivx
+* Mastadon: https://mastodon.social/@pivxcrypto/
```
Comment (JSKitty) [inline 06.community/article.md]: Typo
```suggestion
* Mastodon: https://mastodon.social/@pivxcrypto/
```

**Review by JSKitty — CHANGES_REQUESTED** [review]

**Review by Liquid369 — APPROVED** [review]
ACK 30900fabe8fb972ebd62e906ec2aba006130627d

**Review by JSKitty — APPROVED** [review]
ACK

**Pull request merged**